run_model_on_snellius/exe/build_stemmus_scope_exe.sh

```diff
@@ -7,20 +7,20 @@
 #SBATCH -t 00:05:00
 #SBATCH -N 1
 #SBATCH --ntasks=1
-#SBATCH --cpus-per-task=32
-#SBATCH -p thin
+#SBATCH --cpus-per-task=16
+#SBATCH -p genoa
 #SBATCH --output=./slurm_%j.out
 #SBATCH --error=./slurm_%j.out
 
 # Some security: stop script on error and undefined variables
 set -euo pipefail
 
-############ Use module MATLAB/2021a-upd3 to either run the source code or build the executable file ############
+############ Use module MATLAB/2023a-upd4 to either run the source code or build the executable file ############
```

Comment:
can you also update the matlab version in the readme https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/main/run_model_on_snellius/exe/README.md ?